nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -432,7 +432,17 @@ private int getPage(final Response getResponse, final URL url, final ProcessCont
                     Map<String, String> attributes = new HashMap<>();
                     for(Iterator<Entry<String, JsonNode>> it = source.fields(); it.hasNext(); ) {
                         Entry<String, JsonNode> entry = it.next();
-                        attributes.put(ATTRIBUTE_PREFIX + entry.getKey(), entry.getValue().asText());
+                        String text_value = "";
+                        String separator = "";
```

Comment:
I think it might be better if the value of `separator` didn't change. What do you think about adding each item's text to an array and then doing something like a `StringUtils.join()` on it?